plasmapy/formulary/tests/test_collisions.py

```diff
@@ -1698,3 +1700,347 @@ def test_kwarg_method_error(self):
+                        * T_a ** (-1),
+                        6.8
+                        * 10**-8
+                        * mu_prime ** (1 / 2)
+                        * mu ** (-1)
+                        * T_b ** (-1 / 2)
+                        * T_a ** (-1),
+                    ]
+                )
+            elif limit_type == "fast":
+                limit_values.extend(
+                    [
+                        9
+                        * 10**-8
+                        * (1 / mu + 1 / mu_prime)
+                        * mu ** (1 / 2)
+                        * T_a ** (-3 / 2),
+                        1.8 * 10**-7 * mu ** (-1 / 2) * T_a ** (-3 / 2),
+                        9
+                        * 10**-8
+                        * mu ** (1 / 2)
+                        * mu_prime ** (-1)
+                        * T_b
+                        * T_a ** (-5 / 2),
+                    ]
+                )
+
+        limit_values.append(
+            2 * frequencies.slowing_down.value
+            - frequencies.transverse_diffusion.value
+            - frequencies.parallel_diffusion.value
+        )
+
+        return limit_values
+
+    @pytest.mark.parametrize(
+        "interaction_type, limit_type, constructor_arguments, constructor_keyword_arguments",
+        [
+            # Slow limit (x << 1)
+            (
+                "e|e",
+                "slow",
+                (Particle("e-"), Particle("e-")),
+                {
+                    "T_a": 1 * u.eV,
+                    "T_b": 1e4 * u.eV,
+                    "n_b": 1e15 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                "e|i",
+                "slow",
+                (Particle("e-"), Particle("Na+")),
+                {
+                    "T_a": 1e-4 * u.eV,
+                    "T_b": 1e4 * u.eV,
+                    "n_b": 1e20 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                "i|e",
+                "slow",
+                (Particle("Na+"), Particle("e-")),
+                {
+                    "T_a": 1 * u.eV,
+                    "T_b": 1e2 * u.eV,
+                    "n_b": 1e10 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                "i|i",
+                "slow",
+                (Particle("Na+"), Particle("Cl-")),
+                {
+                    "T_a": 1e2 * u.eV,
+                    "T_b": 1e4 * u.eV,
+                    "n_b": 1e20 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            # Fast limit (x >> 1)
+            (
+                "e|e",
+                "fast",
+                (Particle("e-"), Particle("e-")),
+                {
```

Comment:
In the fast limit cases you should supply v_a not T_a. If the plasma isn't thermal than it doesn't really have a Maxwellian 'temperature' so it's better to avoid using one.